practice-two/src/components/ProductCard.jsx

```diff
@@ -10,12 +9,14 @@ import FavoriteButton from './common/Button/ButtonFavorite';
 /**
  * Product Card Component.
  * @param product - The product data object.
+ * @param isFavoriteProduct - Boolean value indicating if the product is in favorites list.
  * @returns {JSX.Element} Product Card content
  */
-function ProductCard({ product, onSelectFavoriteSelect }) {
+function ProductCard({ product, isFavoriteProduct, onSelectFavorite }) {
```

Comment:
default value

Reply:
Yes, updated default value and comment for this props